extensions/amp-story-interactive/amp-story-interactive.md

```diff
@@ -127,6 +127,24 @@ Results can feed its state from quizzes if all categories also specify `option-{
 </amp-story-interactive-results>
 [/sourcecode]
 
+### amp-story-interactive-slider
+
+The `amp-story-interactive-slider` element provides a voting experience with 101 options. The options take the form of values along a horizontal line, where all options are valid. When selected, an average indicator and the average response are displayed.
```

Comment:
nit: I wonder if `based on a range` is more generally descriptive than `101 options`. Since the user doesn't have to configure 101 options it may be good for the language to be more general.